src/core_modules/capture-core/components/Pages/common/TEIRelationshipsWidget/RegisterTei/RegistrationSection/ProgramSelector/ProgramSelector.component.tsx

```diff
@@ -1,21 +1,20 @@
-// @flow
-import * as React from 'react';
-import { withStyles } from '@material-ui/core/styles';
+import React from 'react';
+import { withStyles, type WithStyles, type Theme } from '@material-ui/core/styles';
 import i18n from '@dhis2/d2-i18n';
 import { ComposedProgramSelector } from './ComposedProgramSelector.component';
+import type { ProgramSelectorProps } from './ProgramSelector.types';
 
 const getStyles = (theme: Theme) => ({
     programLabel: {
         paddingTop: '10px',
-        [theme.breakpoints.down(523)]: {
+        color: theme.palette.text.primary,
+        '@media (max-width: 523px)': {
```

Comment:
Remove the color and revert the breakpoint to be: `[theme.breakpoints.down(523)]: {`